helpers/gsn-balance.js

```diff
@@ -1,21 +0,0 @@
-const program = require('commander')
```

Comment:
* I will not attempt to salvage the git history of this particular change because I literally have something better to do
* I will attempt to cherish the git history of all files moving forward